### DISCUSSION TITLE
Get the value at one of the children

### DISCUSSION THREAD

**Discussion by Harold7665** [atoti/atoti#674] [Q&A] [answered]
Hello,
Is there a way to do this?
![Capture](https://user-images.githubusercontent.com/7359377/186454518-26447589-7342-492e-9493-bc712618a9ad.PNG)
As you can see, from a parent member, I want to take the value of the first child.

**Comment by mailys-ds** [comment, marked as answer]
Hi Harold,
 
Getting the value at the first children is not yet possible with atoti. 
We do have a private method `_first()` based on ActivePivot `first`, see [documentation](https://activeviam.com/activepivot/latest/docs/copper/copper_measures/#navigation-analytic-function), which works on a single level hierarchy.
However, it would require some extra devs on our side to make it an aggregation measure in order to partition by levels on multilevel hierarchies.

**Reply by Harold7665** [reply]
Hello @mailys-ds, thank you!

**Comment by Harold7665** [comment]
I have found this workaround (I am using many single-level hierarchies instead of one many-levels hierarchy):
![workaround](https://user-images.githubusercontent.com/7359377/187168875-fb34fb4a-02d7-437a-83b5-54bf18ea6858.PNG)
